lib/rspecq/reporter.rb

```diff
@@ -9,10 +9,10 @@ module RSpecQ
   #
   # Reporters are readers of the queue.
   class Reporter
-    def initialize(build_id:, timeout:, redis_host:)
+    def initialize(build_id:, timeout:, redis_host:, fail_fast:)
```

Comment:
Since this is not a strictly required option, we should instead make it an attr_accessor and not a parameter of `#initialize`, like `Worker` does for its options.

Reply:
940dc6c

Reply:
That commit is irrelevant I believe.

Reply:
Ignore my comment here - since we're initializing `Queue` in `Reporter#initialize`, what I propose isn't possible.